consumer/pdata/log.go

```diff
@@ -96,9 +96,8 @@ func (ld Logs) LogRecordCount() int {
 	return logCount
 }
 
-// SizeBytes returns the number of bytes in the internal representation of the
-// logs.
-func (ld Logs) SizeBytes() int {
+// Size returns the number of bytes in the internal representation of the logs.
+func (ld Logs) Size() int {
```

Comment:
@tigrannajaryan is it better to name this `OtlpProtoSize` to be consistent with `ToOtlpProtoBytes`. Maybe in the future we will add equivalent for JSON.

Reply:
Yes, I think OtlpProtoSize is a better name.

Reply:
Will do in a separate PR.